README.md

```
@@ -1,2 +1,125 @@
+
+
+<img width="5760" height="3240" alt="Image" src="https://github.com/user-attachments/assets/6cbeb050-50df-4f5c-9084-721ab8689100" />
+
+> **스매싱**은 **2030 세대의 스포츠 자아 완성을 돕는 게이미키케이션 기반** 스포츠 매칭 플랫폼 입니다.
+
+<br/>
+
+## 🏸주요 기능
+
+<img width="590" height="831" alt="Image"
+src="https://github.com/user-attachments/assets/5a24f127-4690-4b6a-bd92-f149c91edd9b" /
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)